### ISSUE TITLE
"Nearby Address" field on 3rd step/panel of Add Tree process blank

### ISSUE THREAD

**Issue by andrewbt** [OpenTreeMap/otm-core#1227]
This appears to happen whether the tree's location was set by clicking on the map and dropping the pin, or by typing in a street address in the Step 1 panel (and moving the tree a bit to satisfy the "Please move marker to exact location of the tree" requirement). "Use Current Location" GPS feature untested.

![add_a_tree_bug_report](https://f.cloud.github.com/assets/2013555/2505792/a57464fa-b39d-11e3-8af6-8a21f13a9792.png)

Steps to reproduce:
1. Hit Add a Tree button on a map to start the process.
2. Set the tree's location by dropping the placemark somewhere; or typing in the address box and selecting one of the geocoded suggestions, and moving the tree a tad. Hit next.
3. Enter Species/DBH info if desired or just hit next.
4. Notice "Nearby Address" is blank on Step 3 of 3 panel.

Suggested functionality:

This field could display the geocoded address selected in step 1. If the placemark was just dropped, or GPS was used, and no geocoding/address normalization was performed, this field could either:

a) disappear entirely, including "Nearby Address" title
b) remain blank as it is now (the user didn't input an address, why should we force trees to have them)
c) a reverse-geocode of the nearest likely address could be performed and added to the tree's info. (whereever it was placed, there's probably a highly likely reverse geocode candidate)

In other words, it doesn't seem useful to perform geocoding/address normalization and not save the resulting correct address with the tree.
